go/oasis-test-runner/cmd/root.go

```diff
@@ -494,6 +497,12 @@ func doScenario(childEnv *env.Env, sc scenario.Scenario) (err error) {
 		}
 	}
 
+	// Populate metrics settings.
+	if viper.IsSet(cfgMetricsAddr) {
+		net.Config().Metrics.Address = viper.GetString(cfgMetricsAddr)
```

Comment:
The statement below implies that `net` could be nil here (although this is probably never the case in practice as all scenarios use fixtures), so this whole section should be moved into that if body below.

Reply:
Oops, fixed :)